docs/BacwardCompatability.md

```diff
@@ -0,0 +1,51 @@
+# Compatability Requirements and Implementation Plan
+
+## What is Compatability
+Before talking about how to provide compatability, it is worthwhile to clarify what we mean by compatibility. 
+Fybrik required to provide forwards and backwards compatibility in its behaviour. Compatibility is hard, especially 
+handling issues around rollback-safety. 
+Let's try to define compatability in this way:
+- Existing clients need not be aware of a new change in order for them to continue to function as they did previously, 
+even when the change is in use.
+- Any Fybrik operations that succeeded before the new change must succeed after the change.
+- Any Fybrik operations that does not use the new change must behave the same as it did before the change.
+- Any Fybrik operations that use the new change must not cause problems (e.g. crash or degrade behavior) when issued 
+against Fybrik objects, e.g. Modules, Plotters, Blueprints and Applications created before the functionality was added.
```

Comment:
FybrikModules, FybrikApplications

Reply:
ok, now we return to the discussion about inconsistent names of fybrik objects